src/object.rs

```diff
@@ -1291,6 +1293,10 @@ pub trait ObjectExt: ObjectType {
     fn find_property<'a, N: Into<&'a str>>(&self, property_name: N) -> Option<::ParamSpec>;
     fn list_properties(&self) -> Vec<::ParamSpec>;
 
+    fn set_qdata(&self, key: Quark, value: Box<dyn Any>);
```

Comment:
Instead of just `Any` this needs to be `Any + Send + Sync` as a general `glib::Object` can be used from arbitrary threads

Reply:
Done